e2e/atlas/clusters_test.go

```diff
@@ -16,22 +16,25 @@
 package atlas_test
 
 import (
+	"crypto/rand"
 	"encoding/json"
 	"fmt"
-	"math/rand"
```

Comment:
gosec is now picky about it, we had no problem using this but changing to a "secure" option was not more complicated than using the safer option